src/backend/renderer/element/mod.rs

```diff
@@ -116,20 +116,20 @@ pub struct RenderElementState {
     /// Holds the visible portion of the element on the output
     ///
     /// Note: If the presentation_state is [`RenderElementPresentationState::Skipped`] this will be zero.
-    pub visible_portion: Size<i32, Physical>,
+    pub visible_portion: usize,
```

Comment:
I get, that we don't need a Rectangle (or multiple) here, but could be add a doc comment explaining, why this is an usize and what exactly it represents?

Reply:
Sure, I will add a comment explaining what it represents.

Reply:
done